main.cpp

```diff
@@ -22,38 +22,43 @@
 ******************************************************************************/
 
 /*!
- * @brief This file contains the main function for the stand-alone well index calculator executable.
+ * @brief This file contains the main function for the stand-alone
+ * well index calculator executable.
  */
 
 #include "main.hpp"
 #include "wellindexcalculator.h"
 #include <Reservoir/grid/eclgrid.h>
 
 using namespace std;
+using Eigen::Vector3d;
```

Comment:
Removed it altogether, it doesn't seem to be needed at all in this file, maybe it was a hold over for something I was trying...